src/redoc/2.3/index.md

```diff
@@ -25,7 +25,7 @@ Each of the following links lead to a list of REST endpoints specific to a user
 
 ## Asynchronous endpoints
 
-You can run POST and PUT endpoints asynchronously while the `async.operations.all` message queue consumer is active. [Asynchronous web endpoints]({{page.baseurl}}/rest/asynchronous-web-endpoints.html) provides information about asynchronous routes, payloads, and responses.
+You can run POST, PUT, and DELETE endpoints asynchronously while the `async.operations.all` message queue consumer is active. [Asynchronous web endpoints]({{page.baseurl}}/rest/asynchronous-web-endpoints.html) provides information about asynchronous routes, payloads, and responses.
```

Comment:
Should we add PATCH request as well? The PATCH, POST, PUT, DELETE requests are supposed to be supported by async API. @nuzil might help us with this decision.

Reply:
Yes, @rogyar is correct. PATCH requests are also supported by Async API, despite there are no usage in the core of Magento, I would add this here as well, in respect to 3rd party integrtions

Reply:
@rogyar @nuzil I added PATCH.